goodtablesio/integrations/github/blueprint.py

```diff
@@ -265,3 +222,43 @@ def api_repo(repo_id):
         'repo': repo,
         'error': error,
     }), code)
+
+
+@github.route('/api/repo/<repo_id>/activate')
+@login_required
+def api_repo_activate(repo_id):
+    error = None
+    token = current_user.github_oauth_token
+    if not token:
+        error = 'No valid GitHub token found'
+    repo = database['session'].query(GithubRepo).get(repo_id)
+    try:
+        activate_hook(token, repo.owner, repo.repo)
+        repo.active = True
+        database['session'].commit()
+    except Exception as exception:
+        log.exception(exception)
+        error = 'Repo activation error'
+    return jsonify({
+        'error': error,
+    })
+
+
+@github.route('/api/repo/<repo_id>/deactivate')
+@login_required
+def api_repo_deactivate(repo_id):
+    error = None
+    token = current_user.github_oauth_token
+    if not token:
+        error = 'No valid GitHub token found'
+    repo = database['session'].query(GithubRepo).get(repo_id)
```

Comment:
I've badly ported it. For S3 I've control flow to check error but missed it here